build/tasks/test-on-node.js

```diff
@@ -59,24 +59,24 @@ module.exports = function( grunt ) {
 		var runDone = false;
 		var testActive = false;
 
-		QUnit.begin( function() {
+		QUnit.on( "runStart", function() {
 			grunt.log.ok( "Testing " + file + " ..." );
 		} );
-		QUnit.testStart( function() {
+		QUnit.on( "testStart", function() {
 			testActive = true;
 		} );
-		QUnit.log( function( details ) {
+		QUnit.on( "assert", function( details ) {
```

Comment:
Personally, I don't find it a good idea, because then everyone could add something. make a little change etc.

Also that's why we have introduced the test `assertions` property on the `testEnd` event , which is specifically designed for this.